server/core/region.go

```diff
@@ -474,11 +486,29 @@ func (r *RegionInfo) GetBuckets() *metapb.Buckets {
 	return (*metapb.Buckets)(buckets)
 }
 
+// GetStorePeerApproximateSize returns the approximate size of the peer on the specified store.
+func (r *RegionInfo) GetStorePeerApproximateSize(storeID uint64) int64 {
+	peer := r.GetStorePeer(storeID)
+	if storeID != 0 && peer != nil && peer.IsWitness {
```

Comment:
Seem `peer != nil` involves `storeID != 0`.

Reply:
I found that some tests will panic, so I added a null pointer judgment